### PR TITLE
Revise Italian code of conduct localization

### PR DESCRIPTION
**Pull request by sftim**
This might well not be ready for merge. I don't speak Italian very well, but I thought I would have a go as it's a short document.

I'll be very happy to get feedback from people who actually speak Italian and then mark this as no longer work-in-progress.

/kind cleanup
Relevant to #18383

### CODE REVIEW

File: content/it/community/code-of-conduct.md [hunk 1]
```diff
@@ -8,17 +8,18 @@ css: /css/community.css
...
+Kubernetes segue il
 <a href="https://github.com/cncf/foundation/blob/master/code-of-conduct.md">codice di condotta CNCF</a>.
-Il testo del CNC CoC è replicato di seguito a partire dal 
-<a href="https://github.com/cncf/foundation/blob/0ce4694e5103c0c24ca90c189da81e5408a46632/code-of-conduct.md">commit 0ce4694</a>.
-Se noti che questo non è aggiornato, ti preghiamo di far presente questo problema.
-<a href="https://github.com/kubernetes/website/issues/new">file an issue</a>.
+Il testo del codice di condotta CNCF è replicato di seguito a partire dal
+<a href="https://github.com/cncf/foundation/blob/214585e24aab747fb85c2ea44fbf4a2442e30de6/code-of-conduct.md">commit 214585e</a>.
+Se noti che questo non è aggiornato, ti preghiamo
+<a href="https://github.com/kubernetes/website/issues/new">inviare un rapporto</a>
```
Comment (fabriziopandini) [inline content/it/community/code-of-conduct.md]: rapporto --> issue
I think that translating terms that are so broadly used might create more confusion than benefits, wdyt?

File: content/it/community/code-of-conduct.md [hunk 2]
```diff
@@ -8,17 +8,18 @@ css: /css/community.css
...
 
-Se noti una violazione del Codice di condotta in occasione di un evento o una riunione, in Slack o in un altro meccanismo di comunicazione, 
-contatta il Comitato per  
-<a href="https://git.k8s.io/community/committee-code-of-conduct">il codice di condotta di Kubernetes/a>. 
-Potete raggiungerci via email all'indirizzo <a href="mailto:conduct@kubernetes.io">conduct@kubernetes.io</a>.
+Se noti una violazione del codice di condotta in occasione di un evento
+o una riunione, in Slack o in un altro meccanismo di comunicazione,
```
Comment (fabriziopandini) [inline content/it/community/code-of-conduct.md]: meccanismo --> canale (oppure mezzo, strumento)

**Comment by sftim** [comment]
/retitle Revise Italian code of conduct localization

**Comment by sftim** [comment]
OK, so I think @fabriziopandini this is ready for review.

[Deploy preview](https://5e165d7d07e065000939df98--kubernetes-io-master-staging.netlify.com/it/community/code-of-conduct/) is **missing** because _it_ localization is currently disabled.

File: content/it/community/code-of-conduct.md [hunk 3]
```diff
@@ -8,17 +8,18 @@ css: /css/community.css
...
+Kubernetes segue il
 <a href="https://github.com/cncf/foundation/blob/master/code-of-conduct.md">codice di condotta CNCF</a>.
-Il testo del CNC CoC è replicato di seguito a partire dal 
-<a href="https://github.com/cncf/foundation/blob/0ce4694e5103c0c24ca90c189da81e5408a46632/code-of-conduct.md">commit 0ce4694</a>.
-Se noti che questo non è aggiornato, ti preghiamo di far presente questo problema.
-<a href="https://github.com/kubernetes/website/issues/new">file an issue</a>.
+Il testo del codice di condotta CNCF è replicato di seguito a partire dal
+<a href="https://github.com/cncf/foundation/blob/214585e24aab747fb85c2ea44fbf4a2442e30de6/code-of-conduct.md">commit 214585e</a>.
+Se noti che questo non è aggiornato, ti preghiamo
+<a href="https://github.com/kubernetes/website/issues/new">inviare un issue</a>
```
Comment (fabriziopandini) [inline content/it/community/code-of-conduct.md]: un --> una

Comment (sftim) [reply content/it/community/code-of-conduct.md]: Tweaked & pushed

**Comment by fabriziopandini** [comment]
@sftim are we going to merge this on master or on the italian branch?

**Comment by sftim** [comment]
I'm fine targeting either. What do you recommend?

**Comment by fabriziopandini** [comment]
Let's keep all the work on it branch until we get critical mall

**Comment by sftim** [comment]
@fabriziopandini please take a look: I think this is ready to merge in.

**Comment by fabriziopandini** [comment]
Sorry for the delayed review
/lgtm
/approve

**Comment by sftim** [comment]
Thanks @fabriziopandini 
Prow: anything I need to do here?

**Pull request merged**